src/GitVersionCore/IncrementStrategyFinder.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace GitVersion
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using VersionCalculation.BaseVersionCalculators;
+    using LibGit2Sharp;
+
+    public enum CommitMessageVersionMode
```

Comment:
I think you will need to attribute up the Enum members with YmlMember() attributes. Let me try and find some docs